coordinator/provider/coordinator.go

```diff
@@ -432,7 +442,12 @@ func (qc *qdbCoordinator) AddKeyRange(ctx context.Context, keyRange *kr.KeyRange
 		})
 
 		if err != nil {
-			return err
+			spqrlog.Zero.Debug().
+				Str("router", r.ID).
+				Err(err).
+				Msg("etcdqdb: notify router add key range")
+			continue
+			//return err
```

Comment:
why?

Reply:
I was not sure here. As I see the code, we could use `traverseRouters` to notify about new key ranges, and in that function we notify all the routers even if error occurred somewhere. But here we wanted to stop if any error raised